test/functional/normalization/json_format_test_elastic.go

```diff
@@ -0,0 +1,139 @@
+package normalization
+
+import (
+	"encoding/json"
+	"github.com/ViaQ/logerr/log"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers/types"
+	"github.com/openshift/cluster-logging-operator/test/matchers"
+	"github.com/openshift/cluster-logging-operator/test/runtime"
+	"strings"
+)
+
+var _ = Describe("[Functional][Outputs][ElasticSearch] FluentdForward Output to ElasticSearch", func() {
+
+	const (
+		elasticSearchTag   = "7.10.1"
+		elasticSearchImage = "elasticsearch:" + elasticSearchTag
+
+		// json message
+		jsonMsg = "{\\\"name\\\":\\\"fred\\\",\\\"home\\\":\\\"bedrock\\\"}"
+		jsonErrorMsg = "{\\\"name\\\":\\\"fred\\\",\\\"home:\\\"bedrock\\\"}"
+	)
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+
+		newVisitor = func(f *functional.FluentdFunctionalFramework) runtime.PodBuilderVisitor {
+			return func(b *runtime.PodBuilder) error {
+				log.V(2).Info("Adding ElasticSearch output container", "name", logging.OutputTypeElasticsearch)
+				b.AddContainer(logging.OutputTypeElasticsearch, elasticSearchImage).
+					AddEnvVar("discovery.type", "single-node").
+					AddRunAsUser(2000).
+					End()
+				return nil
+			}
+		}
+
+	)
+
+	BeforeEach(func() {
+
+		framework = functional.NewFluentdFunctionalFramework()
+		functional.NewClusterLogForwarderBuilder(framework.Forwarder).
+			FromInput(logging.InputNameApplication).
+			ToElasticSearchOutput()
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+	Context("when sending json to ElasticSearch "+elasticSearchTag +" and require parsing ", func() {
+		It("should parse correctly and send to custom index ( json-index ) ", func() {
```

Comment:
`json_index` is not a custom idex. it is hardcoded

Reply:
sure ... this (as you state correctly) come from customers using API / CRD